lib/katello/plugin.rb

```diff
@@ -668,6 +668,13 @@ def katello_template_setting_values(name)
         default: false,
         full_name: N_('Hide Reclaim Space Warning'),
         description: N_('If this is enabled, the Smart Proxy page will suppress the warning message about reclaiming space.')
+
+      setting 'deb_enable_structured_apt',
```

Comment:
Since Satellite doesn't support deb, should probably add this to https://github.com/RedHatSatellite/foreman_theme_satellite/blob/develop/app/models/concerns/upstream_only_settings.rb once this is merged.

Reply:
Should I open an PR for that? Note also that the setting is not intended to be kept for ever. Eventually, there will come a Katello release where all users are migrated to structured APT during upgrade. After that the setting can  be dropped from the code.

Reply:
Dropped.